tests/functional/functionaltesting.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+##################################################################################
+#   Tests that should pass
+##################################################################################
+$tests = array(
+ 	"google global" => array(
+    "request" => array(
+      'region'=>'in',
+          'search_engine' => 'google',
+           'phrase' => 'আপনি এক কাপ চা বা কফি এক কাপ বা নরম পানীয় একটি কাপ চান?',
+		   'language' => 'bn',
+			)
+        ),
+);
+
+##################################################################################
+#   Tests that should return an api error message about max results being over 500
+##################################################################################
+$errormaxresults = array(
+    
+);
+
+// Uncommoment the following lines and add the proper values
+ define('API_KEY', 'We4vJk3fWvm-2SPq');
```

Comment:
this is an huge security risk,

please remove credentials, maybe better to have them in settings.php file as the file is ignored